### ISSUE TITLE
Migrate to PyTorch 1.2

### ISSUE THREAD

**Issue by ymoisan** [NRCan/geo-deep-learning#77]
Pytorch 1.1.0 is now released : https://github.com/pytorch/pytorch/releases/tag/v1.1.0.  Would there be some use in the new TensorBoard integration for [Hypertrainer](https://github.com/lemairecarl/hypertrainer) ?

**Comment by ymoisan** [comment]
Travis CI already uses 1.1.  All we have to do is modify our own dev/test environments.

**Comment by mpelchat04** [comment]
Changes in Pytorch 1.2.0:  
`
/conda_envs/env_name/lib/python3.6/site-packages/torch/optim/lr_scheduler.py:82: 
`
UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule.See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
  "https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate", UserWarning)
